docs/docs/Contributing/contributing-templates.md

```diff
@@ -0,0 +1,89 @@
+---
+title: Contribute templates
+slug: /contributing-templates
+---
+
+Follow these best practices when submitting a template to Langflow.
+
+For template formatting examples, see the Langflow repository's [starter_projects](https://github.com/langflow-ai/langflow/tree/main/src/backend/base/langflow/initial_setup/starter_projects) folder.
+
+## Create a PR to submit your template
+
+Follow these steps to submit your template:
+
+1. Fork the Langflow repository on GitHub.
+2. Add your `template.json` file to the `src/backend/base/langflow/initial_setup/starter_projects` folder in your fork.
+3. Create a Pull Request from your fork to the main Langflow repository.
+4. Include a screenshot of your template in the PR.
+5. The Langflow team will review your PR, offer feedback, and merge the template.
+
+## Name
```

Comment:
I'm not sure these items are worthy of having individual headings. Also, you don't introduce them, so at first I was wondering why there were headings with titles like "Name" and "Description".